external/maybe-sync-external.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview
```

Comment:
Please fill in, esp explain when "maybe-sync" means "don't sync"

Reply:
+1